P5/Source/Specs/att.locatable.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright TEI Consortium. 
+Dual-licensed under CC-by and BSD2 licences 
+See the file COPYING.txt for details
+$Date$
+$Id$
+-->
+<?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+<classSpec xmlns="http://www.tei-c.org/ns/1.0" module="tei" type="atts" ident="att.locatable">
+  <desc versionDate="2020-01-15" xml:lang="en">provides attributes that specify locational information.</desc>
+  <attList xmlns:rng="http://relaxng.org/ns/structure/1.0">
+    <attDef ident="where" usage="opt">
+      <desc versionDate="2020-01-15" xml:lang="en">The <att>where</att> specifies a location by using a URI to another resource.</desc>
```

Comment:
maybe better: "by using a URI to point at another resource"?

Reply:
The wording is a little weird here too: "The `@where` specifies...". Should be "the `<att>where</att>` attribute specifies...", no?

Reply:
Thanks--just checking these off as I edit them. :-)